src/containers/Pipelines/Pipelines.js

```diff
@@ -91,6 +94,11 @@ export /* istanbul ignore next */ class Pipelines extends Component {
                           {pipelineName}
                         </Link>
                       </StructuredListCell>
+                      <StructuredListCell>
+                        <Link to={`/pipelines/${pipelineName}`}>
```

Comment:
we should have a tooltip on this to identify its purpose, also helpful for accessibility

same for Tasks